@hallfox  Test error on `x64-linux` locally:
```
Installing 4/4 ntf-core:x64-linux...
Building ntf-core:x64-linux...
-- Downloading https://github.com/bloomberg/ntf-core/archive/2.1.0.tar.gz -> bloomberg-ntf-core-2.1.0.tar.gz...
-- Extracting source /home/test/vcpkg/downloads/bloomberg-ntf-core-2.1.0.tar.gz
-- Using source at /home/test/vcpkg/buildtrees/ntf-core/src/2.1.0-b5bd59a649.clean
-- Configuring x64-linux
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:112 (message):
    Command failed: /home/test/vcpkg/downloads/tools/ninja/1.10.2-linux/ninja -v
    Working Directory: /home/test/vcpkg/buildtrees/ntf-core/x64-linux-rel/vcpkg-parallel-configure
    Error code: 1
    See logs for more information:
      /home/test/vcpkg/buildtrees/ntf-core/config-x64-linux-dbg-CMakeCache.txt.log
      /home/test/vcpkg/buildtrees/ntf-core/config-x64-linux-rel-CMakeCache.txt.log
      /home/test/vcpkg/buildtrees/ntf-core/config-x64-linux-out.log

 

Call Stack (most recent call first):
  installed/x64-linux/share/vcpkg-cmake/vcpkg_cmake_configure.cmake:252 (vcpkg_execute_required_process)
  ports/ntf-core/portfile.cmake:9 (vcpkg_cmake_configure)
  scripts/ports.cmake:147 (include)
```

`/home/test/vcpkg/buildtrees/ntf-core/config-x64-linux-out.log`:
```
CMake Error at repository.cmake:3916 (find_program):
  Could not find debugger_path using the following names: gdb
Call Stack (most recent call first):
  targets.cmake:18 (ntf_repository)
  CMakeLists.txt:42 (include)
```

--- a/ports/ntf-core/portfile.cmake
+++ b/ports/ntf-core/portfile.cmake
@@ -1,0 +1,53 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO bloomberg/ntf-core
+    REF "${VERSION}"
+    SHA512 57662d2dd105b2781e580623c26cd7bde84fce8374bbd70c18595a5f6934869b7a570f0d3c2e17e115f6c7eb1067541f8d19523639815b285324061f807d3179
+    HEAD_REF main
+)
+
+# ntf-core requires debugger information to for dev tooling purposes, so we just fake it
+vcpkg_cmake_configure(
+    SOURCE_PATH "${SOURCE_PATH}"
+    OPTIONS "-DNTF_BUILD_WITH_USAGE_EXAMPLES=0"
+            "-DNTF_TOOLCHAIN_DEBUGGER_PATH=NOT-FOUND"
+)
+
+vcpkg_cmake_build()
+
+vcpkg_cmake_install()
+
+function(fix_pkgconfig_ufid lib_dir ufid pc_name)
+    vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/${lib_dir}/pkgconfig/${pc_name}.pc" "/${ufid}" "")
+    if ("${ufid}" MATCHES opt)
+        set(build_mode "release")
+    else()
+        set(build_mode "debug")
+    endif()
+
+    vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/${lib_dir}/cmake/${pc_name}Targets-${build_mode}.cmake" "/${ufid}" "")
+endfunction()
+
+function(fix_install_dir lib_dir ufid)
+    message(STATUS "Fixing ufid layout for ${CURRENT_PACKAGES_DIR}/${lib_dir}/${ufid}")
+    file(RENAME "${CURRENT_PACKAGES_DIR}/${lib_dir}/${ufid}" "${CURRENT_PACKAGES_DIR}/tmp")
+    file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/${lib_dir}")
+    file(RENAME "${CURRENT_PACKAGES_DIR}/tmp" "${CURRENT_PACKAGES_DIR}/${lib_dir}")
+
+    fix_pkgconfig_ufid("${lib_dir}" "${ufid}" "nts")
+    fix_pkgconfig_ufid("${lib_dir}" "${ufid}" "ntc")
+endfunction()
+
+fix_install_dir("lib" "opt_exc_mt")
+fix_install_dir("debug/lib" "dbg_exc_mt")
+vcpkg_cmake_config_fixup(CONFIG_PATH "lib/cmake")
+
+# Handle copyright
+vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/LICENSE")
+vcpkg_fixup_pkgconfig()
+
+# Usage
+file(INSTALL "${CMAKE_CURRENT_LIST_DIR}/usage" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}")
+
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
+

--- a/ports/ntf-core/usage
+++ b/ports/ntf-core/usage
@@ -1,0 +1,9 @@
+The package ntf-core provides the CMake targets:
+
+    # Blocking and non-blocking sockets for network programming
+    find_package(nts REQUIRED)
+    target_link_libraries(main PRIVATE nts)
+
+    # Asynchronous sockets, timers, event loops, and thread pools for network programming
+    find_package(ntc REQUIRED)
+    target_link_libraries(main PRIVATE ntc)

--- a/ports/ntf-core/vcpkg.json
+++ b/ports/ntf-core/vcpkg.json
@@ -1,0 +1,17 @@
+{
+  "name": "ntf-core",
+  "version": "2.1.0",
+  "description": "The Network Transport Framework: Core Libraries",
+  "license": "Apache-2.0",
+  "dependencies": [
+    "bde",
+    {
+      "name": "vcpkg-cmake",
+      "host": true
+    },
+    {
+      "name": "vcpkg-cmake-config",
+      "host": true
+    }
+  ]
+}

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -5764,6 +5764,10 @@
       "baseline": "2019-08-10",
       "port-version": 3
     },
+    "ntf-core": {
+      "baseline": "2.1.0",
+      "port-version": 0
+    },
     "nu-book-zxing-cpp": {
       "baseline": "2.1.0",
       "port-version": 0

--- a/versions/n-/ntf-core.json
+++ b/versions/n-/ntf-core.json
@@ -1,0 +1,9 @@
+{
+    "versions": [
+        {
+            "version": "2.1.0",
+            "port-version": 0,
+            "git-tree": "eabeb2b30205bef45e72e4a069f21d71797f3f5d"   
+        }
+    ]
+}

